mediasms.go

```diff
@@ -0,0 +1,114 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/comvex-jp/mediasms-go/errors"
+	"github.com/comvex-jp/mediasms-go/models"
+	"github.com/comvex-jp/mediasms-go/translations"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// resultCode from media4u
+type resultCode struct {
+	Result string `json:"result"`
+}
+
+func createSMSID(prefix, messageID string) string {
+	return prefix + messageID
+}
+
+// SMSURL constant for sms-console
+const SMSURL = "https://www.sms-console.jp/"
+
+// Send request to media4u
+func (c Client) Send(messageID string, val models.SendRequest) (models.APIResponse, error) {
+	smsID := createSMSID(c.Prefix, messageID)
+
+	val.SMSID = smsID
+
+	jsonValue, _ := json.Marshal(val)
+	httpClient := &http.Client{}
+
+	req, _ := http.NewRequest("POST", SMSURL+"api/", bytes.NewBuffer(jsonValue))
+	req.SetBasicAuth(c.Username, c.Password)
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := httpClient.Do(req)
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Isn't this going to stop the process?
I wonder if a library should be logging Fatal at all, as it is out of control of the library consumer.

Reply:
No idea. Do you have a suggestion?

Reply:
Perhaps return an appropriate error message instead, not too sure to be honest.
`log.Fatal` seems to execute `os.Exit(1)`.

Reply:
A couple of options. We could return an empty response, and the error. e.g. `return models.APIResponse{}, err` Then have the error be handled in the `API`. Other choice is to define a new response ourselves. 
e.g. 
```
return models.APIResponse{
                StatusCode:  500,
		Name:        "httpClient error",
		Description: "Error occurred when making an API call",
```

Or something to that effect.